src/components/Typography/Caption/Caption.tsx

```diff
@@ -1,38 +1,72 @@
 import * as React from "react";
 import { HasComponent } from "../../../types";
-import { usePlatform } from "../../../hooks/usePlatform";
 import { classNames } from "../../../lib/classNames";
-import { getClassName } from "../../../helpers/getClassName";
+import { warnOnce } from "../../../lib/warnOnce";
+import { Footnote } from "../Footnote/Footnote";
+import { resolveWeight } from "../../../helpers/typography";
 import "./Caption.css";
 
 export interface CaptionProps
   extends React.AllHTMLAttributes<HTMLElement>,
     HasComponent {
-  weight: "regular" | "medium" | "semibold" | "bold";
-  level: "1" | "2" | "3" | "4";
+  /**
+   * Начертания "bold", "semibold", "medium" и "regular" устарели и будут удалены в 5.0.0. Используйте значения "1", "2" и "3"
+   */
+  weight?: "regular" | "medium" | "semibold" | "bold" | "1" | "2" | "3";
+  /**
+   * Уровень "4", "semibold", "medium" и "regular" устарел и будет удален в 5.0.0. Используйте компонент Footnote
```

Comment:
Уровни остаются такими же, как и были, только 4 уровень упразднен в пользу Footnote.

```suggestion
   * Уровень "4" устарел и будет удален в 5.0.0. Используйте компонент Footnote
```